src/app/ApplicationTemplate.Shared.Views/App.xaml

```diff
@@ -12,9 +12,11 @@
 				<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
 				
 				<!-- Uno material -->
-				<MaterialColorsV1 xmlns="using:Uno.Material"
+				<MaterialFonts xmlns="using:Uno.Material"
+							   OverrideSource="ms-appx:///Styles/Application/Fonts.xaml" />
```

Comment:
I see that you set the fonts override, but the file hasn't changed. Does that mean that the names are already the ones needed for the material override?

Reply:
they are in the common folder from the Uno Material... meaning that they're suposed to work for V1 and V2
https://github.com/unoplatform/Uno.Themes/blob/master/src/library/Uno.Material/Styles/Application/Common/Fonts.xaml
however, they're using the old naming (MaterialXXXXX), that's why i did the override to keep using the fonts already in the app

Reply:
I don't think V1 allowed to inject a different font into the material styles?
In our case, we want to override Roboto in favor of Guillon.

Reply:
ok, so we're good though, since we're using our Fonts(Guillon), what i could do is to remove all references to Roboto locally

Reply:
Yes you should be able to remove all references to Roboto. It's supposed to be bundled in the Uno.Material nuget package anyways.

Reply:
But I want to make sure that the fonts override actually works.
I don't understand how the override would currently work knowing that the fonts are named as `"FontFamily01XXX"`

Reply:
https://platform.uno/docs/articles/external/uno.themes/doc/material-getting-started.html#change-default-font

Reply:
because our fonts file has that naming convention **FontFamily01XXX**, 
also in this documentation says to keep using this **MaterialFonts** on the App.xaml instead of MaterialTheme 
[MaterialFonts ](https://platform.uno/docs/articles/external/uno.themes/doc/material-migration.html#continue-using-v1-styles)

Reply:
Is the Guillon font properly applied on material buttons?

Reply:
they're now :p

Reply:
You didn't setup the fonts override correctly.

What does this mean?
> because our fonts file has that naming convention FontFamily01XXX,

**We don't need** that naming convention.

Reply:
so, are we keeping the Material as Prefix?? the override for the Fonts are wrong??

Reply:
Done